chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/activityMiddleware.spec.ts

```diff
@@ -60,15 +66,15 @@ describe("activityMiddleware test", () => {
             }
         };
 
-        const results = createActivityMiddleware()()(next)(args);
+        const results = createActivityMiddleware(renderMarkdown)()(next)(args);
         expect(TelemetryHelper.logActionEvent).toHaveBeenCalledTimes(0);
-        expect(results().props?.dangerouslySetInnerHTML?.__html).toEqual(systemMessage);
+        expect(results().props?.dangerouslySetInnerHTML?.__html).toContain("system message");
     });
 
     it ("createActivityMiddleware() should escape html texts to prevent XSS attacks", () => {
         spyOn(TelemetryHelper, "logActionEvent").and.callFake(() => false);
         const next = (args: any) => () => args; // eslint-disable-line @typescript-eslint/no-explicit-any
-        const systemMessage = "<img src='' onerror=\"alert('XSS attack')\"/>";
+        const systemMessage = "<img src='' onerror=\"alert('XSS attack')\"/></img>";
```

Comment:
this is an already closed tag, no need of the closing it again

Reply:
removed </img> tag